.github/workflows/jira-issue-label-added.yaml

```diff
@@ -9,5 +9,5 @@ jobs:
   call-workflow:
     uses: stoplightio/.github/.github/workflows/jira-issue-label-added.yaml@master
     with:
-      team-name: ${{ github.event.label.name == 'team/bear-claws' && 'Bear Claws' || github.event.label.name == 'team/honey-hackers' && 'Honey Hackers'}}
+      team-name: "Bear Claws"
```

Comment:
I admit I'm a little confused here.  It looks like "Bear Claws" will always be added now, rather than a team name dependent on the original event label.  Is that correct?  (Maybe I just haven't been keeping up with this ticket.)

Reply:
Yes I'm sorry I didn't add more context here.  I broke the action with the current team-name logic, so I'm replacing it with "Bear Claws" for now to test that a new Jira task will be created with the correct team name.  Also, I wanted to get it back to a working state until I figure out how to set the team name based on the label.